src/components/audit/row.vue

```diff
@@ -19,19 +19,22 @@ except according to the terms contained in the LICENSE file.
       </template>
     </td>
     <td class="initiator">
-      <actor-link v-if="audit.actor != null" :actor="audit.actor"/>
+      <template v-if="audit.actor != null">
+        <span v-if="audit.actor.deletedAt != null" class="icon-trash" :title="$t('deletedMessage')"></span>
+        <actor-link :actor="audit.actor"/>
```

Comment:
The white space between the `<span>` and the `<actor-link>` is automatically trimmed, so to add space between them, I use `$margin-right-icon` below. (I often do that with icons.) I also tweaked the Target column to do something similar.